docs/installation/mac.md

```diff
@@ -232,7 +317,7 @@ Work through this section to try some practical container tasks using `boot2dock
 
 ### Mount a volume on the container
 
-When you start `boot2docker`, it automatically shares your `/Users` directory
+When you start a container it automatically shares your `/Users` directory
```

Comment:
Not sure if this will be in the coming release, but I heard this will be more strict in a coming release, and only `/Users/<your-username>/` will be shared.

@ehazlett should be able to confirm this.